sbt-dumpschema-plugin/src/test/scala/org/virtuslab/akkasaferserializer/SchemaWriterSpec.scala

```diff
@@ -0,0 +1,40 @@
+package org.virtuslab.akkasaferserializer
+
+import better.files.File
+import io.bullet.borer.Json
+import org.scalatest.matchers.should
+import org.scalatest.wordspec.AnyWordSpecLike
+import org.virtuslab.akkasaferserializer.model.{ClassAnnotation, Codecs, Field, TypeDefinition}
+
+class SchemaWriterSpec extends AnyWordSpecLike with should.Matchers with Codecs {
+  val testDef: TypeDefinition =
+    TypeDefinition(isTrait = false, "test", Seq("anno"), Seq(Field("a", "Int")), Seq("one", "two"))
+
+  "SchemaWriter" should {
+
+    "load previous dump" in {
```

Comment:
Hmmm do we even need access to **previous** dump? can't the entire dump be generated in a single go for the sake of  simplicity/better reproducibility? 🤔

Reply:
Unfortunately, no. I have to consider incremental compilation.